core/src/main/java/org/sonarsource/sonarlint/core/container/connected/update/QualityProfilesDownloader.java

```diff
@@ -58,6 +62,7 @@ public void fetchQualityProfiles(Path destDir) {
       throw new IllegalStateException("Failed to load default quality profiles", e);
     }
 
-    ProtobufUtil.writeToFile(qProfileBuilder.build(), destDir.resolve(StorageManager.QUALITY_PROFILES_PB));
+    QProfiles build = qProfileBuilder.build();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "build". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1488)